### PR TITLE
This is my Comp 110 essay proposal

### PR DESCRIPTION
**Pull request by xvii-tommy**
I could not compile the latex file to pdf.
i also am yet to find all of the papers online.

### CODE REVIEW

File: proposal/COMP110 architecture essay/comp110 essay proposal.tex [hunk 1]
```diff
@@ -0,0 +1,72 @@
+\documentclass{article}
+
```
Comment (edpowley) [inline proposal/COMP110 architecture essay/comp110 essay proposal.tex:2]: To make the document compile, you need to insert the following line here:

``` latex
\usepackage[hidelinks]{hyperref}
```

File: proposal/COMP110 architecture essay/comp110 essay proposal.tex [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+\item[web link:] \url{http://ieeexplore.ieee.org/xpl/login.jsp?tp=&arnumber=7004920&url=http%3A%2F%2Fieeexplore.ieee.org%2Fxpls%2Fabs_all.jsp%3Farnumber%3D7004920}
+\item[Full text link:]\url{http://floppsie.comp.glam.ac.uk/Papers/paper21/2d-game-engine-mulley.pdf}
+\item[Comments:] i found this paper which is about predictive collision in 2d game engines. This paper looks into the construction of an engine which uses predictive collision over a frame based approach which means collisons are calculated in advance.
+\end{description}
+
+\section*{Paper 2}
+
+\begin{description}
+\item[Title:] An Improved Algorithm of Collision Detection in 2D Grapple Games
+\item[Citation:]\cite{<guo2010improved>}
```
Comment (Adrir) [inline proposal/COMP110 architecture essay/comp110 essay proposal.tex:29]: :+1:

File: proposal/COMP110 architecture essay/comp110 essay proposal.tex [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+\item[web link: ] \url{http://ieeexplore.ieee.org/xpl/login.jsp?tp=&arnumber=5453586&url=http%3A%2F%2Fieeexplore.ieee.org%2Fxpls%2Fabs_all.jsp%3Farnumber%3D5453586}
+\item[Full text link:]
+\item[Comments:]This paper is about automaticaly detecting inteference between two objects it provides an algorithm which detects collisions exactly an sees if objects vertexes are enclosed.
+\end{description}
+
+\section*{Paper 3} 
+
+\begin{description}
+\item[Title:] Distance computation using axis aligned bounding box (AABB) parallel distribution of dynamic origin point
+\item[Citation:]\cite{<sulaiman2013distance>}
```
Comment (Adrir) [inline proposal/COMP110 architecture essay/comp110 essay proposal.tex:40]: :+1:

File: proposal/COMP110 architecture essay/comp110 essay proposal.tex [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+\item[web link:] \url{http://ieeexplore.ieee.org/xpl/login.jsp?tp=&arnumber=6575955&url=http%3A%2F%2Fieeexplore.ieee.org%2Fxpls%2Fabs_all.jsp%3Farnumber%3D6575955}
+\item[Full text link:]
+\item[Comments:]this paper introduces a new technique which looks for the closest distance between two objects. this can make colision detection more effective.
+\end{description}
+
+\section*{Paper 4}
+
+\begin{description}
+\item[Title:] Interactive Mesostructures withVolumetric Collisions
+\item[Citation:]\cite{<nykl2014interactive>}
```
Comment (Adrir) [inline proposal/COMP110 architecture essay/comp110 essay proposal.tex:51]: This algorithm requires a GPU and associated data structure. Unless you are familiar with these concepts, I suggest avoiding them at this time.

File: proposal/COMP110 architecture essay/comp110 essay proposal.tex [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+\item[web link:]\url{http://ieeexplore.ieee.org/xpl/articleDetails.jsp?arnumber=6799298}
+\item[Full text link:]
+\item[Comments:]This paper looks into a new technique for interactively coliding with and deforming mesostructures at a par text level. it is a new algorithm which can be used with existing physics engines and new games so it can be a more effective physics engine for games.it allows for accumulating deformations without the need for new geomatry. 
+\end{description}
+
+\section*{Paper 5}
+
+\begin{description}
+\item[Title:] Collision course by transformation of coordinates and plane decomposition
+\item[Citation:]\cite{<bendjilali2009collision>}
```
Comment (Adrir) [inline proposal/COMP110 architecture essay/comp110 essay proposal.tex:62]: Looks interesting -- unfortunately, I do not have access to Cambridge University Press.

File: proposal/COMP110 architecture essay/comp110 essay proposal.tex [hunk 6]
```diff
@@ -0,0 +1,72 @@
...
+\maketitle
+
+\section*{Topic}
+My essaye will be on Convexity-based collision detection for a 2D game engine.
+
+\section*{Paper 1}
+
+\begin{description}
+\item[title:] The Construction of a Predictive Collision 2D Game Engine
+\item[Citation:]\cite{<mulley2013construction>}
```
Comment (Adrir) [inline proposal/COMP110 architecture essay/comp110 essay proposal.tex:18]: A technique is described, but it is more sophisticated that collision detection - in that it models the predicted time of collision of two objects with given velocities. Something a bit less sophisticated is recommended.

**Comment by Adrir** [comment]
Consider looking up another primary source -- I suggest looking for GJK.

**Pull request closed without merging**